build.sbt

```diff
@@ -680,9 +680,20 @@ val generateRustParserLib =
   ) {
     val os = System.getProperty("os.name")
     if (os.startsWith("Mac")) {
-        Seq("cargo", "build", "-p", "enso-parser-jni", "--target", "x86_64-apple-darwin") !
+      Seq(
+        "cargo",
+        "build",
+        "-p",
+        "enso-parser-jni",
+        "--target",
+        "x86_64-apple-darwin",
+        "-Z",
+        "unstable-options",
+        "--out-dir",
+        "target/rust/debug"
```

Comment:
This shouldn't be hard-coded. Instead one should use the value of the output target setting.